homeassistant/components/inels/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""The iNels integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from inelsmqtt import InelsMqtt
+from inelsmqtt.devices import Device
+from inelsmqtt.discovery import InelsDiscovery
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import (
+    BROKER,
+    BROKER_CONFIG,
+    COORDINATOR_LIST,
+    DOMAIN,
+    LOGGER,
+    STARTUP_MESSAGE,
+)
+from .coordinator import InelsDeviceUpdateCoordinator
+
+PLATFORMS: list[Platform] = [
+    Platform.SWITCH,
+    Platform.LIGHT,
+    Platform.COVER,
+    Platform.SENSOR,
+]
+
+
+async def _async_config_entry_updated(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Call when config entry being updated."""
+
+    client: InelsMqtt = hass.data[BROKER]
+
+    await hass.async_add_executor_job(client.disconnect)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up iNels from a config entry."""
+    if CONF_HOST not in entry.data:
+        LOGGER.error("MQTT broker is not configured")
+        return False
+
+    LOGGER.info(STARTUP_MESSAGE)
```

Comment:
No other integration does this as far as I know. Please remove it. The start of the integration is already logged.

Reply:
@fabaff removed